core/circuitbreaker/slot.go

```diff
@@ -5,7 +5,8 @@ import (
 )
 
 const (
-	RuleCheckSlotName = "sentinel-core-circuit-breaker-rule-check-slot"
+	RuleCheckSlotName  = "sentinel-core-circuit-breaker-rule-check-slot"
+	RuleCheckSlotOrder = 5000
```

Comment:
It is better to define the order of slots in a uniform location, which may seem more intuitive。

Reply:
My first version is like what you recommended, current version is the result of discussion with @louyuting .